### DISCUSSION TITLE
Add a message that zpub or ypub are converted to xpub

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#2163] [General]
I've noticed on numerous occasions that people get confused that we convert ypub and zpub to xpub. Often times they'd wonder if what they've added makes sense.

We need to find a better way to communicate this. Any ideas on how we can better communicate that we convert this?

https://user-images.githubusercontent.com/36959754/102879081-89322b80-4449-11eb-85f3-fd44dd8bfd90.mp4

Example of a confused user adding zpub.

![Screenshot from 2020-12-22 11-33-53](https://user-images.githubusercontent.com/36959754/102879128-9d762880-4449-11eb-8e43-9ba35fbc0ccf.png)

**Comment by Kukks** [comment]
We should be showing the original "xpub" along with our internal converted version IMO

**Reply by pavlenex** [reply]
For user internal converted version doesn't really mean a thing though? Hide it and show what they're entering makes sense as well.

**Reply by Kukks** [reply]
Hmm I think you're right but we will need to experiment with it and see how it feels. If we do indeed have a bug with the conversion, the UI will be lying to the user and the issue would never be findable

**Comment by NicolasDorier** [comment]
> We should be showing the original "xpub" along with our internal converted version IMO

This is currently already the case. Here is an example of a multisig imported from Sparrow's output descriptor.
The wallet settings is showing what the user has input (`Derivation Scheme Input`), but as well as the NBXplorer format. (`Derivation Scheme`)

![image](https://github.com/user-attachments/assets/850e47fd-5fa1-4f61-86f3-399fb690453a)
